### PR TITLE
Added image_path stuff

### PR DESCRIPTION
**Pull request by callumacrae**
This allows us to use `imagePath('cat.jpg')`, which will then turn into its revisioned filename.

### CODE REVIEW

File: test/browserify.js [hunk 1]
```diff
@@ -376,4 +377,54 @@ describe('browserify', function () {
       done();
     });
   });
+
+  describe('image_path', function () {
+    // This horrible hack is to stop is being deleted straight away!
```
Comment (djgrant) [inline test/browserify.js:382]: Please fix this comment

File: test/browserify.js [hunk 2]
```diff
@@ -376,4 +377,54 @@ describe('browserify', function () {
...
+    beforeEach(function (done) {
+      fs.writeFile('rev-manifest.json', '{"cat.jpg":"cat-123.jpg"}', function (err) {
+        done(err);
+      });
+    });
+
+    it('should handle image_path correctly', function (done) {
```
Comment (djgrant) [inline test/browserify.js]: this means NOTHING @callumacrae

**Comment by djgrant** [comment]
GREAT WORK

File: test/browserify.js (hunk 2 above)
Comment (callumacrae) [inline test/browserify.js]: dont care

File: test/browserify.js (hunk 1 above)
Comment (callumacrae) [inline test/browserify.js:382]: no

**Pull request merged**